ansible/roles/monitoring/defaults/main.yml

```diff
@@ -1 +1,7 @@
 enable_log_ingestion: false
+
+# ECS monitoring 
+ecs_aws_region: eu-central-1
+monitoring_secret_key: "{{lookup('amazon.aws.aws_ssm', '/oonidevops/secrets/ooni_monitoring/secret_key', profile='oonidevops_user_dev')}}"
```

Comment:
Do we maybe need to separate between the prod and dev access keys? I see you are just looking up the dev ones for the moment, but eventually we might need one for too no?

Reply:
Yeah you're right, I didn't added the prod version because the monitoring Terraform module is not currently used in the prod environment. Do you think I should add it? For now it only adds a new user for monitoring and some access keys, see: https://github.com/ooni/devops/blob/main/tf/modules/ooni_monitoring/main.tf

Reply:
Let's keep it like this and change it only when we need it